components/meta-packages/history/history

```diff
@@ -534,12 +534,15 @@ desktop/virt-manager@0.6.1,5.11-2015.0.2.0
 developer/build/automake-19@1.9.6,5.11-5.12.0.0.0.16.0
 developer/build/cbe@1.8.1,5.11-2017.0.0.1
 developer/clang-3.3@3.3,5.11-2015.0.1.2
+developer/clang-34@3.4.2-2017.0.0.6
```

Comment:
sorry, I've missed it. You removed history from components, but didn't add it to meta-packages/history. Please, add developer/clang-3.4  and developer/clang-3.8 here at the same versions as they are in history files.